### DISCUSSION TITLE
Grapher multiple columns plot

### DISCUSSION THREAD

**Discussion by gemGreg** [adamerose/pandasgui#191] [Q&A] [answered]
In the old version of PandasGUI there was a way to plot multiple signals on one line plot.
![image](https://user-images.githubusercontent.com/7453447/151777966-afad982e-08a3-4a62-b623-edb8456558ce.png)
Is there a way to do it in the newest version of PandasGUI? 
It seems like In the newest version you can drag/drop only one signal to X and Y:
![image](https://user-images.githubusercontent.com/7453447/151778169-80e10534-58e8-43b4-b1d7-af4bdf22e6cd.png)

**Comment by adamerose** [comment, marked as answer]
Yes this is an accidental regression but should be easy to fix. Created an issue

**Comment by jjYBdx4IL** [comment]
not yet fixed in 0.2.13 on windows

**Comment by DeyoSwed** [comment]
Not fixed with 0.2.13 on window 11

**Comment by Cortexico** [comment]
This was fixed here #192
